spyderlib/baseconfig.py

```diff
@@ -204,18 +205,102 @@ def get_image_path(name, default="not_found.png"):
 #==============================================================================
 # Translations
 #==============================================================================
+LANG_FILE = osp.join(get_conf_path(), '.langconfig')

```

Comment:
I'm ok with adding this new file, but don't call it `.langconfig`, just `langconfig`, so that it's not hidden on Linux :-)

At the end, it's going to be inside of `~/.spyder`, so there is no need to be hidden.